src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenAsyncTests.vb

```diff
@@ -8323,8 +8323,13 @@ Class C
         Yield o
     End Function
     Shared Sub Main()
-        For Each o in M()
-            Console.WriteLine(o)
+        For Each e As Exception in M()
+            ' Cannot just call .ToString() on the exception, because the exact format of a stack trace depends on a localization
+            Console.WriteLine($"{e.GetType()}: {e.Message}")
+            For Each frame In New Diagnostics.StackTrace(e).GetFrames()
+                Dim m = frame.GetMethod()
+                Console.WriteLine($"   at {m.DeclaringType.FullName.Replace("+"c, "."c)}.{m.Name}({String.Join(",", DirectCast(m.GetParameters(), Object()))})")
```

Comment:
On Windows with Russian language pack, the string representation of a stack trace uses Russian preposition "в" instead of English "at", so I format it manually.